examples/hello/tasks/gatewayCall.ts

```diff
@@ -0,0 +1,126 @@
+import { task, types } from "hardhat/config";
+import type { HardhatRuntimeEnvironment } from "hardhat/types";
+import ZRC20ABI from "@zetachain/protocol-contracts/abi/ZRC20.sol/ZRC20.json";
+
+const main = async (args: any, hre: HardhatRuntimeEnvironment) => {
+  const { ethers } = hre;
+  const [signer] = await ethers.getSigners();
+
+  const txOptions = {
+    gasPrice: args.txOptionsGasPrice,
+    gasLimit: args.txOptionsGasLimit,
+  };
+
+  const revertOptions = {
+    abortAddress: "0x0000000000000000000000000000000000000000", // not used
+    callOnRevert: args.callOnRevert,
+    onRevertGasLimit: args.onRevertGasLimit,
+    revertAddress: args.revertAddress,
+    revertMessage: ethers.utils.hexlify(
```

Comment:
there's any helper where we can add encodeReverMessage as hexlify + toUtf?

Reply:
We can make a helper for this, but it will basically just wrap two functions. I didn't want to introduce a dependency just for this.

Reply:
I agree, if the helper was there yes but just for this makes no sense